modules/nf-core/shinyngs/validatefomcomponents/main.nf

```diff
@@ -8,12 +8,16 @@ process SHINYNGS_VALIDATEFOMCOMPONENTS {
         'quay.io/biocontainers/r-shinyngs:1.5.1--r42hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(sample), path(feature_meta), path(assay_files)
-    tuple val(meta2), path(contrasts)
+    tuple val(meta),  path(sample)
```

Comment:
Can we leave the samples and assays in the same input channel? That makes sense I think

Reply:
done!

Reply:
actually, I am just now wondering if there is a reason why you added the meta2 to contrasts? The metamap is always the same, isn't it? If so I would just have it input into the module in the first input channel, and the other input is just provided without the `tuple val(meta)` thing?

Reply:
No, these are representing different things, the contrast meta has different content to the samples/assays meta.

Reply:
In the specific test examples the meta is the same, but I wouldn't want to impose that assumption.